### DISCUSSION TITLE
how can users specify command line arguments to julia

### DISCUSSION THREAD

**Discussion by FelipeLema** [Olical/conjure#386] [Q&A] [answered]
hey, y'all

great plugin going on here, I really appreciate the effort being made.

I've got a question (or feature request): how can one specify command line arguments to the julia command? I'm looking to add `--project=.` to load some project-level packages (think "packages from current python's virtualenv")

thanks again

**Comment by glyh** [comment, marked as answer]
Maybe you can set this variable, `vim.g['conjure#client#julia#stdio#command']`. It defaults to`julia --banner=no --color=no -i`

There's no project level support yet I guess, but you could write some codes to tweak around it/ contribute it to conjure.

**Reply by Olical** [reply]
There's also stuff like https://github.com/Olical/nvim-local-fennel

Which can let you run config based on your current project or directory tree. There are LOADS of plugins out there that do this, I've just linked my one that's Fennel based but there's lots of .vimrc ones out there if people prefer.
